client/apps/hello_world/app.jsx

```diff
@@ -45,7 +47,13 @@ if (window.DEFAULT_JWT) { // Setup JWT refresh
 const mainApp =  document.getElementById('main-app');
 initResizeHandler(mainApp);
 
-ReactDOM.render(
-  <Root store={store} />,
-  mainApp,
-);
+ltiLaunch(window.DEFAULT_SETTINGS).then((valid) => {
+  if (valid) {
+    ReactDOM.render(
+      <Root store={store} />,
+      mainApp,
+    );
+  } else {
+    document.body.innerHTML = 'Invalid request. Please reload the page.';
+  }
+});
```

Comment:
We could also make the validation into a react component.  Then error handling and accessing settings would just work.